homeassistant/components/openweathermap/repairs.py

```diff
@@ -0,0 +1,31 @@
+"""Issues for OpenWeatherMap."""
+
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers import issue_registry as ir
+
+from .const import DOMAIN
+
+
+def _get_issue_id(entry_id: str) -> str:
+    return "deprecated_v25_" + entry_id
+
+
+@callback
+def async_create_issue(hass: HomeAssistant, entry_id: str) -> None:
+    """Create issue for V2.5 deprecation."""
+    ir.async_create_issue(
+        hass=hass,
+        domain=DOMAIN,
+        issue_id=_get_issue_id(entry_id),
+        is_fixable=False,
```

Comment:
We usually try not to have non-fixable issues.  I think the fix here is to switch the options to 3.0

Reply:
done